manifests/08-prometheus_rule.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: insights-prometheus-rules
+  namespace: openshift-insights
+spec:
+  groups:
+  - name: insights
+    rules:
+    - alert: InsightsDisabled
+      annotations:
+        description: 'Insights operator is disabled. In order to enable Insights and benefit from recommendations specific to your cluster, please follow steps listed in the documentation: https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/enabling-remote-health-reporting.html'
```

Comment:
This link is not accurate and has been moved, and points to a page that doesn't resolve. 

```
$ curl -IL https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/enabling-remote-health-reporting.html
HTTP/2 301 
date: Mon, 06 Dec 2021 13:17:09 GMT
server: Apache/2.4.34 (Red Hat) OpenSSL/1.0.2k-fips
location: http://docs.openshift.com/container-platform/4.9/support/remote_health_monitoring/enabling-remote-health-reporting.html
cache-control: max-age=1
expires: Mon, 06 Dec 2021 13:17:10 GMT
content-type: text/html; charset=iso-8859-1
set-cookie: 0bad6f3ecddb231c72e656b5cff74d2d=b2db64e4bd6a20b23bd7d187492bb56d; path=/; HttpOnly; Secure; SameSite=None
cache-control: private

HTTP/1.1 302 Found
content-length: 0
location: https://docs.openshift.com/container-platform/4.9/support/remote_health_monitoring/enabling-remote-health-reporting.html
cache-control: no-cache

HTTP/2 404 
date: Mon, 06 Dec 2021 13:17:09 GMT
server: Apache/2.4.34 (Red Hat) OpenSSL/1.0.2k-fips
accept-ranges: bytes
content-length: 10801
x-ua-compatible: IE=Edge,chrome=1
content-type: text/html; charset=utf-8
set-cookie: 0bad6f3ecddb231c72e656b5cff74d2d=b2db64e4bd6a20b23bd7d187492bb56d; path=/; HttpOnly; Secure; SameSite=None
cache-control: private
```

Reply:
As mentioned in the PR template. This requires https://github.com/openshift/openshift-docs/pull/39518